journald/lib_journald.hpp

```diff
@@ -18,10 +18,105 @@ namespace journald {
 
 class JournaldContainerLoggerProcess;
 
-struct Flags : public virtual flags::FlagsBase
+
+// These flags are loaded twice: once when the `ContainerLogger` module
+// is created and each time before launching executors. The flags loaded
+// at module creation act as global default values, whereas flags loaded
+// prior to executors can override the global values.
+struct LoggerFlags : public virtual flags::FlagsBase
```

Comment:
I'd argue that the fully qualified name (`mesos::journald::LoggerFlags`) is sufficient to say that this doesn't apply to all logs.